PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/utility/AccessPrivate.java

```diff
@@ -1,4 +1,4 @@
-package nl.tudelft.pl2016gr2.test.utility;
```

Comment:
Your commit where you change the package naming does not contain a description. Because of that, I do not know what your reason for this rename was.

What is wrong with it? In that layer (the pl2016gr2 folder) it is normal to specify subfolders based on what its contents are for (these utilities are for tests only)